bindings/wasm/examples/src/utils_web.js

```diff
@@ -1,17 +1,19 @@
-import * as identity from "../../web/identity_wasm.js";
+import * as identity from "@iota/identity-wasm";
+
+export const LINK_REGEX = /(\b(https?|ftp):\/\/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/gim;
 
 /**
  * Loads the identity wasm library.
  *
  * @returns {Promise<void>}
  */
-export async function initIdentity(path = "../../web/identity_wasm_bg.wasm", log = true) {
-    if(log) logToScreen("Initialization started...");
+export async function initIdentity(path = "./identity_wasm_bg.wasm") {
+    console.log("Initialization started...");
     try {
         await identity.init(path);
-        if(log) logToScreen("Initialization success!");
+        console.log("Initialization success!");
     } catch (err) {
-        if(log) logToScreen(err);
+        throw new Error(err);
```

Comment:
redundant try catch?

Reply:
good point, will remove the try catch and add sourcemaps